api_core/google/api_core/grpc_helpers.py

```diff
@@ -308,8 +310,8 @@ def stream_stream(
 
     def subscribe(self, callback, try_to_connect=False):
         """grpc.Channel.subscribe implementation."""
-        return
+        pass
```

Comment:
With a docstring you don't need any function body (if it pleases you)

Reply:
Noted.